utilities/unittests/test_storage.py

```
@@ -169,3 +169,35 @@ class TestConstructDatavolumeSourceDictUnsupported:
     def test_unsupported_source_raises_value_error(self):
         with pytest.raises(ValueError, match="Unsupported source type: ftp"):
             construct_datavolume_source_dict(source="ftp")
+
+
+class TestAssertVolumeReady:
+    @pytest.fixture()
```

Ruff (0.16.0)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)